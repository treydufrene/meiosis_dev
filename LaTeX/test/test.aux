\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\providecommand \oddpage@label [2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.0.1}\protected@file@percent }
\newlabel{sec:intro}{{1}{1}{Introduction}{section.0.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Specifications and Tests}{1}{section.0.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {3}Conclusion}{1}{section.0.3}\protected@file@percent }
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Summary of Test Results\relax }}{1}{table.caption.1}\protected@file@percent }
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{tab:results}{{1}{1}{Summary of Test Results\relax }{table.caption.1}{}}
\newlabel{sec:app}{{3}{2}{Appendix}{section*.2}{}}
\bibstyle{plainnat}
\bibdata{robo}
