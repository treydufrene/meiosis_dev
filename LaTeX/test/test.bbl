\begin{thebibliography}{5}
\providecommand{\natexlab}[1]{#1}
\providecommand{\url}[1]{\texttt{#1}}
\expandafter\ifx\csname urlstyle\endcsname\relax
  \providecommand{\doi}[1]{doi: #1}\else
  \providecommand{\doi}{doi: \begingroup \urlstyle{rm}\Url}\fi

\bibitem[arm()]{arm}
{"Cortex-A53," arm Developer}.

\bibitem[gri({\natexlab{a}})]{gripper1}
{\natexlab{a}}.
\newblock URL
  \url{https://www.trossenrobotics.com/p/phantomx-parallel-ax12-gripper.aspx?feed=Froogle&gclid=EAIaIQobChMIl_6Wsu3B6AIVJoNaBR3l3A_2EAQYASABEgK1o_D_BwE}.

\bibitem[gri({\natexlab{b}})]{gripper2}
{\natexlab{b}}.
\newblock URL
  \url{https://www.trossenrobotics.com/productdocs/assemblyguides/phantomx-parallel-gripper.html}.

\bibitem[mit()]{mit}
{"The MIT License," Open Source Initiative}.
\newblock URL \url{https://opensource.org/licenses/MIT}.

\bibitem[rpi()]{rpi}
{"Raspberry Pi 3 Model B+"}.

\end{thebibliography}
