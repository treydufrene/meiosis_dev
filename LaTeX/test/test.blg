This is BibTeX, Version 0.99d (TeX Live 2019)
Capacity: max_strings=100000, hash_size=100000, hash_prime=85009
The top-level auxiliary file: test.aux
The style file: plainnat.bst
Database file #1: robo.bib
Warning--entry type for "mit" isn't style-file defined
--line 11 of file robo.bib
Warning--entry type for "arm" isn't style-file defined
--line 16 of file robo.bib
Warning--entry type for "rpi" isn't style-file defined
--line 22 of file robo.bib
Warning--entry type for "gripper2" isn't style-file defined
--line 34 of file robo.bib
Warning--entry type for "gripper1" isn't style-file defined
--line 40 of file robo.bib
Warning--to sort, need author or key in gripper1
Warning--to sort, need author or key in gripper2
Warning--to sort, need author or key in mit
Warning--to sort, need author or key in rpi
Warning--to sort, need author or key in arm
Warning--empty year in arm
Warning--empty year in gripper1
Warning--empty year in gripper2
Warning--empty year in mit
Warning--empty year in rpi
You've used 5 entries,
            2773 wiz_defined-function locations,
            602 strings with 5303 characters,
and the built_in function-call counts, 1195 in all, are:
= -- 104
> -- 0
< -- 0
+ -- 6
- -- 0
* -- 86
:= -- 96
add.period$ -- 7
call.type$ -- 5
change.case$ -- 10
chr.to.int$ -- 4
cite$ -- 25
duplicate$ -- 75
empty$ -- 176
format.name$ -- 0
if$ -- 285
int.to.chr$ -- 2
int.to.str$ -- 1
missing$ -- 0
newline$ -- 25
num.names$ -- 0
pop$ -- 63
preamble$ -- 1
purify$ -- 10
quote$ -- 0
skip$ -- 81
stack$ -- 0
substring$ -- 15
swap$ -- 7
text.length$ -- 0
text.prefix$ -- 0
top$ -- 0
type$ -- 55
warning$ -- 10
while$ -- 0
width$ -- 0
write$ -- 46
(There were 15 warnings)
