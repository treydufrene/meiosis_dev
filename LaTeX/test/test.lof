\contentsline {figure}{\numberline {1}{\ignorespaces Labeled Image of Manipulator Render\relax }}{1}% 
\contentsline {figure}{\numberline {2}{\ignorespaces Measurement Test for Link Offset\relax }}{2}% 
\contentsline {figure}{\numberline {3}{\ignorespaces Manipulator in its Zeroed Configuration\relax }}{3}% 
\contentsline {figure}{\numberline {4}{\ignorespaces Solidworks Measurement of Reachable Workspace\relax }}{4}% 
\contentsline {figure}{\numberline {5}{\ignorespaces Test of Link Length for Specification 1.5.a.\relax }}{5}% 
\contentsline {figure}{\numberline {6}{\ignorespaces Matlab Plot of Invalid Dexterity Check\relax }}{6}% 
\contentsline {figure}{\numberline {7}{\ignorespaces Matlab Plot of Successful Dexterity Check\relax }}{7}% 
\contentsline {figure}{\numberline {8}{\ignorespaces Dynamixel Parallel Gripper with Servo Modified from \cite {gripper1}\relax }}{8}% 
\contentsline {figure}{\numberline {9}{\ignorespaces Dynamixel Gripper with and without Foam Padding at Minimum and Maximum Opening \cite {gripper1}\relax }}{8}% 
\contentsline {figure}{\numberline {10}{\ignorespaces Dynamixel Gripper Rotational to Linear Motion Mechanism \cite {gripper2}\relax }}{9}% 
\contentsline {figure}{\numberline {11}{\ignorespaces GUI Template\relax }}{11}% 
