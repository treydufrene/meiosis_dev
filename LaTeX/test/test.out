\BOOKMARK [1][-]{section.0.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.0.2}{Specifications and Tests}{}% 2
\BOOKMARK [1][-]{section.0.3}{Conclusion}{}% 3
